src/main/js/Widgets/AJAXDialog.js

```diff
@@ -27,11 +27,13 @@ class AJAXDialog {
   /**
    * Constructs a new dialog box, which is dynamically built and then populated with the HTML returned from an AJAX call.
    *
+   * @param {boolean} dismissible Whether dialog is easily dismissible by clicking outside dialog or pressing escape. Defaults to true.
    * @constructor
    */
-  constructor() {
+  constructor(dismissible = true) {
```

Comment:
We should set this parameter using the options configuration. 

See `_setInitialOptions()`. One example will be how we set the callback function. See `callback` in the initial options object, and `withCallback` function. 

All widget options will have a default set in `_setInitialOptions()` and can be set using a `with*` function, or the ` withOptions` functions which sets all options at once. 

Some widgets can also specify configuration using `data-` attributes, this is not one of those widgets, but for reference, here is one that does use that strategy as well. 
- https://github.com/inversoft/prime.js/blob/8b2a9b10090c1d20cc12f8cea6ce0f7ced2dfac3/src/main/js/Widgets/DateTimePicker.js#L1022

Reply:
Thanks.  Converted to options.